### DISCUSSION TITLE
Issue with LDAP Plugin for Mirth Connect v3.12

### DISCUSSION THREAD

**Discussion by MaheshBalasubramanian** [nextgenhealthcare/connect#5505] [Q&A]
Hi Team,

We have installed the LDAP Authorization Plugin in Mirth Connect v3.12. As the LDAP plugin was enabled and user groups are added. 
No users are fetched while clicking on "Test Connection" and getting the Error "**No Users were Found**".In the Search Contexts, we are providing this **OU=zzz,OU=NON_PRO,OU=APPS,OU=Groups,DC=xxx,DC=yyy**.  

Could you please let us know how to resolve this issue? 

Regards,
Mahesh B.

**Comment by pacmano1** [comment]
Just log into the server and delete the plugin directory that has the named plugin (I can't recall what that is for the LDAP plugin) and restart mirth.

The LDAP plugin will be in a directory in the "extensions" subdirectory of your Mirth install.
